screens/auth/LoginScreen.js

```diff
@@ -146,6 +148,9 @@ export default class Login extends React.Component {
   render() {
     return (
       <AuthScreenContainer>
+        <BackButton onPress={() => this.props.navigation.goBack(null)}>
+          <FontAwesome5 name="arrow-left" solid size={24} />
+        </BackButton>
```

Comment:
Is this BackButton supposed to be positioned somewhere at the top left of the screen? Fine in iPhone 11 but on iPhone 8 it appears to be closer to "Log in" than the top.
![image](https://user-images.githubusercontent.com/35573990/77873272-0eeac700-727c-11ea-9ae0-e0d8d789be3c.png)

Reply:
This is also adjusted to be higher in #56 for the keyboard covering issues. 👍